helm/cilium/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 0.19.2
-appVersion: 1.14.5
+version: 1.15.1
```

Comment:
Was this intended? The current chart version still is 0.19.2.

Reply:
Thanks, I'll change to 0.20.0

Reply:
as I said before. this will only bring confusion. we need to take a better approach regarding sync: https://github.com/giantswarm/cilium-app/pull/104/files#r1374117353

(of course ignore my comment, I'm not asking you to do anything at this stage 😄 )

Reply:
Maybe to bring that discussion up again: I usually file a sync PR and revert the parts we want to keep manually, especially if they are in files which are actively changed by upstream and can not be overwritten by inheritance (what we are doing in `_gs-helpers.tpl` e.g.).

Reply:
For now https://github.com/giantswarm/cilium-upstream/pull/9